app/src/main/java/com/keylesspalace/tusky/util/ThemeUtils.kt

```diff
@@ -35,7 +35,7 @@ private const val THEME_DAY = "day"
 private const val THEME_BLACK = "black"
 private const val THEME_AUTO = "auto"
 private const val THEME_SYSTEM = "auto_system"
-const val APP_THEME_DEFAULT = THEME_NIGHT
+const val APP_THEME_DEFAULT = THEME_SYSTEM
```

Comment:
Because you're changing the default this will affect everyone who is upgrading, and not just new users. There are many places in the code that do

```
val theme = sharedPreferences.getString("appTheme", APP_THEME_DEFAULT)
```

We have a mechanism to handle this, so that people who are upgrading do not see the UI suddenly change out from under them.

First you will need to update `PrefKeys.SCHEMA_VERSION`, in https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt#L44.

Then you will need to modify `upgradeSharedPreferences()` (https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/TuskyApplication.kt#L121) so that, on upgrade, if the user's current theme preference is not set, it's set to the `THEME_NIGHT` constant.

Reply:
Thank you for your really detailed comments! I thought it was too easy... I do the requested changes and push them somewhere this week or the next one.

Reply:
Done, I hope I have well understand your guidelines :)